src/app/App.tsx

```diff
@@ -70,13 +79,35 @@ const App = () => {
             <Route path="documentation" element={<ComingSoon />} />
             <Route path="audit-report-upload" element={<ReportUpload />} />
             <Route path="/report/:uuid" element={<CertificationResult />} />
+            <Route path="/support-commands" element={<SupportCommands />} />
+            <Route path="/support-commands/:id" element={<SupportCommandDetails />} />
           </Route>
         </Routes>
       </Suspense>
       <CustomGPT />
+
+      {/* Impersonate banner */}
+      {impersonate ? (
+        <div className="inline-flex justify-between fixed top-0 left-0 w-full bg-[#000000bd] z-[10000] p-2 text-slate-300 text-center font-bold">
+          <span className="self-center">Impersonating: {selectedUser.fullName}</span>
+          <span
+            className="inline-flex items-center hover:cursor-pointer hover:underline"
+            title="Stop Impersonating"
+            onClick={async () => {
+              const impersonatedProfile = retainId;
+              await dispatch(setImpersonate({ status: false, id: null }));
+              await dispatch(clearRun());
+              window.location.pathname = `/support-commands/${impersonatedProfile}`;
```

Comment:
For navigation we should use `useNavigate` from `react-router-dom`
Like we do on the [NavBar component](https://github.com/input-output-hk/dapps-certification-web/blob/master/src/pages/session/components/NavBar.tsx#L37)